web/packages/teleterm/src/ui/hooks/useLoggedInUser.ts

```diff
@@ -1,20 +1,52 @@
 import { useAppContext } from 'teleterm/ui/appContextProvider';
+import { useWorkspaceContext } from 'teleterm/ui/Documents';
+import { LoggedInUser } from 'teleterm/services/tshd/types';
 
-export function useLoggedInUser() {
-  const ctx = useAppContext();
+/**
+ * useLoggedInUser returns the logged in user of the root cluster of the currently active workspace.
```

Comment:
```suggestion
 * useLoggedInUser returns the user logged into the root cluster of the active workspace.
```

Maybe simplify the comment?